main.js

```diff
@@ -9,18 +9,32 @@ define(function (require, exports, module) {
         LiveDevServerManager = brackets.getModule("LiveDevelopment/LiveDevServerManager"),
         PreferencesManager   = brackets.getModule("preferences/PreferencesManager"),
         ProjectManager       = brackets.getModule("project/ProjectManager"),
+        LiveDevelopment      = brackets.getModule("LiveDevelopment/LiveDevMultiBrowser"),
+        // PostMessageTransport = require("LiveDevelopment/MultiBrowserImpl/transports/PostMessageTransport"),
```

Comment:
I believe it's up to us. If we wanted to upstream this then I'd argue this is where it should live, but I guess that's up to them in the end. Would you prefer it as part of the extension?